package.json

```diff
@@ -1,29 +1,35 @@
 {
-  "name": "@triniti/admin-ui",
+  "name": "@triniti/admin-ui-plugin",
   "version": "0.1.0",
   "description": "The admin-ui is a javaScript plugin for Triniti apps.",
   "repository": {
     "type": "git",
     "url": "https://github.com/triniti/admin-ui-plugin-js"
   },
-  "license": "UNLICENSED",
-  "private": true,
-  "main": "src/index.jsx",
+  "license": "Apache-2.0",
+  "module": "./index.jsx",
+  "jsnext:main": "./index.jsx",
   "scripts": {
-    "build": "BABEL_ENV=browser webpack -p --env.NODE_ENV=production --progress --profile --colors",
+    "build": "npm run clean && npm run lint && npm run test && npm run build:babel && npm run build:prepub && npm run build:rm-scripts",
+    "build:babel": "BABEL_ENV=es6 babel src --out-dir ./dist",
+    "build:prepub": "cp -rf ./{*.md,package.json} ./dist",
+    "build:rm-scripts": "jq 'del(.babel)' ./package.json | jq 'del(.scripts)' > ./dist/package.json",
+    "clean": "rimraf ./dist",
     "lint": "BABEL_ENV=cjs eslint src tests",
+    "pub": "npm run build && cd dist && npm publish --access public",
     "test": "BABEL_ENV=cjs tape -r ./tests/bootstrap.js './tests/**/*.test.js*' | faucet"
   },
-  "dependencies": {
-    "@gdbots/common": "^0.1.0",
+  "dependencies": {},
+  "peerDependencies": {
+    "@gdbots/common": "^0.1.0 || ^0.2.0 || ^0.3.0 || ^0.4.0 || ^1.0.0",
+    "@triniti/app": "^0.1.0",
```

Comment:
use `"^0.1.0 || ^0.2.0 || ^0.3.0 || ^0.4.0 || ^1.0.0"` constraint for `@triniti/app`